sdk/nodejs/src/cloudClient.ts

```diff
@@ -22,6 +22,15 @@ const castIncomingUser = (user: DVCUser) => {
     return user
 }
 
+const throwIfUserError = (err: AxiosError, isVariableRequst?: boolean) => {
```

Comment:
is the `isVariableRequest` parameter being used?

Reply:
nope, my bad - fixed!